app/models/chat_channel.rb

```diff
@@ -122,6 +122,21 @@ def title_from_chatable
     end
   end
 
+  def email_title(user)
+    if direct_message_channel?
+      usernames = (chatable.users.map(&:username) - [user.username]).map { |username| "@#{username}" }
```

Comment:
Should we sort the list of users like we do in "normal" email notification?

Reply:
We should use the existing `title` method and sort by username there instead. It was recently improved in #859.